src/main/java/org/kohsuke/stapler/idea/language/JellyParserDefinition.java

```diff
@@ -0,0 +1,27 @@
+package org.kohsuke.stapler.idea.language;
+
+import com.intellij.lang.xml.XMLParserDefinition;
+import com.intellij.psi.FileViewProvider;
+import com.intellij.psi.PsiFile;
+import com.intellij.psi.tree.IFileElementType;
+import org.kohsuke.stapler.idea.psi.JellyFileImpl;
+
+/**
+ * Jelly Parser Definition to associate proper types
+ *
+ * @author Julien Greffe
+ */
+public class JellyParserDefinition extends XMLParserDefinition {
+
+    IFileElementType JELLY_FILE = new IFileElementType(JellyLanguage.INSTANCE);
```

Comment:
```suggestion
    public static final IFileElementType JELLY_FILE = new IFileElementType(JellyLanguage.INSTANCE);
```

Reply:
Based on https://plugins.jetbrains.com/docs/intellij/lexer-and-parser-definition.html#define-a-parser